From 277f101011a001408dd39a58553d5317b1d2ebbf Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Wed, 22 Jul 2026 04:12:15 -0400
Subject: [PATCH] winewayland: Change vulkan debug channel to waylanddrv.

(cherry picked from commit fda30836ca29ce4b60ed9bec45ca2b58da7ebfaa)
---
 dlls/winewayland.drv/wayland_surface.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/winewayland.drv/wayland_surface.c b/dlls/winewayland.drv/wayland_surface.c
index 9ce964aa009..6e6712e8cf8 100644
--- a/dlls/winewayland.drv/wayland_surface.c
+++ b/dlls/winewayland.drv/wayland_surface.c
@@ -5310,6 +5310,7 @@ void wayland_client_surface_attach_image_description(struct wayland_client_surfa
             wp_color_management_surface_v1_destroy(surface->wp_color_management_surface_v1);
             surface->wp_color_management_surface_v1 = NULL;
         }
+        wl_display_flush(process_wayland.wl_display);
         return;
     }
 
